maxim/tests/test_langgraph.py

```
@@ -139,47 +134,71 @@ class GraphConfig(TypedDict):
 
+    @langgraph_agent(name="ask_agent", logger=logger)
+    async def ask_agent(self, query: str) -> str:
+        config = {"recursion_limit": 50, "callbacks": [langchain_callback()]}
+        async for event in app.astream(input={"messages": [query]}, config=config):
+            for k, v in event.items():
+                if k == "agent":
+                    response = str(v["messages"][0].content)
+        return response
+
+    @trace(name="chat_v1", id=lambda: str(uuid4()), logger=None)
+    async def handle_chat(self, query: str, req_id: str):
```

Ruff (0.11.9)

Missing return type annotation for public function `handle_chat`

(ANN201)